scipy/constants/_codata.py

```diff
@@ -1495,6 +1532,95 @@
 Wien wavelength displacement law constant                   2.897 771 955... e-3     (exact)                  m K
 W to Z mass ratio                                           0.881 53                 0.000 17                   """
 
+
+def exact2018(exact):
+    # SI base constants
+    c = exact['speed of light in vacuum']
+    h = exact['Planck constant']
+    e = exact['elementary charge']
+    k = exact['Boltzmann constant']
+    N_A = exact['Avogadro constant']
```

Comment:
The original PR defined some useful constants to reduce duplication. I'll add them back.